vispy/color/colormap.py

```diff
@@ -508,7 +510,158 @@ def map(self, t):
+
+    Notes
+    -----
+    For more information about the hue values see the `wikipedia page`_.
+
+    .. _wikipedia page: https://en.wikipedia.org/wiki/Hue
+    """
+
+    def __init__(self, hue=200, saturation_range=[0.1, 0.8], value=1.0):
+        colors = ColorArray([
+            (hue, saturation_range[0], value),
+            (hue, saturation_range[1], value)
+        ], color_space='hsv')
+        super(_SingleHue, self).__init__(colors)
+
+
+class _HSL(Colormap):
+    """A colormap which is defined by n evenly spaced points in
+    a circular color space.
+
+    This means that we change the hue value while keeping the
+    saturation and value constant.
+
+    Parameters
+    ---------
+    n_colors : int, optional
+        The number of colors to generate.
+    hue_start : int, optional
+        The hue start value. Must be in the range [0, 360], the default is 0.
+    saturation : float, optional
+        The saturation component of the colors to generate. The default is
+        fully saturated (1.0). Must be in the range [0, 1.0].
+    value : float, optional
+        The value component of the colors to generate or "brightness". Must
+        be in the range [0, 1.0], and the default is 1.0
+    controls : array-like, optional
+        The list of control points for the colors to generate. It should be
+        an increasing list of floating-point number between 0.0 and 1.0.
+        The first control point must be 0.0. The last control point must be
+        1.0. The number of control points depends on the interpolation scheme.
+    interpolation : str, optional
+        The interpolation mode of the colormap. Default: 'linear'. Can also
+        be 'zero'.
+        If 'linear', ncontrols = ncolors (one color per control point).
+        If 'zero', ncontrols = ncolors+1 (one color per bin).
+    """
+
+    def __init__(self, ncolors=6, hue_start=0, saturation=1.0, value=1.0,
+                 controls=None, interpolation='linear'):
+        hues = np.linspace(0, 360, ncolors + 1, dtype='int')[:-1]
+        hues += hue_start
+        hues %= 359
+
+        colors = ColorArray([(hue, saturation, value) for hue in hues],
+                            color_space='hsv')
+
+        super(_HSL, self).__init__(colors, controls=controls,
+                                   interpolation=interpolation)
+
+
+class _HUSL(Colormap):
+    """A colormap which is defined by n evenly spaced points in
+    the HUSL hue space.
+
+    Parameters
+    ---------
+    n_colors : int, optional
+        The number of colors to generate.
+    hue_start : int, optional
+        The hue start value. Must be in the range [0, 360], the default is 0.
+    saturation : float, optional
+        The saturation component of the colors to generate. The default is
+        fully saturated (1.0). Must be in the range [0, 1.0].
+    value : float, optional
+        The value component of the colors to generate or "brightness". Must
```

Comment:
This means "(The value component or brightness) of the colours to generate.", right? Not '(The value component) or (the string "brightness")'? Because it could be interpreted the second way.